README.md

```diff
@@ -1,2 +1,8 @@
 # Portfolio
+
 A clean and clear portfolio page
+
+# To make the portfolio page
+
+    *using -Html and css
+          -using Vs Code a text editor
```

Comment:
- [ ] Please make your readme file more descriptive by following the given template above